javascripts/govuk/analytics/analytics.js

```diff
@@ -13,6 +13,11 @@
       delete config.universalId
       this.trackers.push(new GOVUK.GoogleAnalyticsUniversalTracker(universalId, config))
     }
+    if (typeof config.govukTrackerGifUrl !== 'undefined') {
```

Comment:
I think this redundant since you switched to using AJAX in the `sendData` method.